src/css/components/three_up.scss

```diff
@@ -22,19 +23,15 @@
     margin-bottom: 1rem;
   }
 
-  &.bt2px {
-    border-top: 2px solid;
-  }
-
-  &.b-red {
+  &.part-3 {
```

Comment:
The convention we've been using in our components is that elements of a component (like this part) are named with the component name so:

`.three_up-part-1`, etc.

Is there a reason you want to do something different here?